### PR TITLE
Change formatting of reports graph legends to million, billion etc

### PR DESCRIPTION
**Pull request by SudharakaP**
Resolve https://github.com/opencollective/opencollective/issues/5021

image

### CODE REVIEW

File: components/host-dashboard/reports-section/helpers.js [hunk 1]
```diff
@@ -5,6 +5,11 @@ export const getActiveYearsOptions = host => {
   return activeYears.map(year => ({ value: year, label: year })).reverse();
 };
 
-export const formatAmountForLegend = value => {
-  return value < 1000 ? value : `${Math.round(value / 1000).toLocaleString()}k`;
+export const formatAmountForLegend = (locale, currency, value) => {
```
Comment (znarf) [inline components/host-dashboard/reports-section/helpers.js:8]: value should stay first argument as it was before, this is what you format, the rest are options.

```suggestion
export const formatAmountForLegend = (value, currency, locale) => {
```

Comment (znarf) [reply components/host-dashboard/reports-section/helpers.js]: It's not acceptable to integrate this change and mark it as "resolved" without updating the places where it's used. I expect that you think about this yourself.

**Review by znarf — CHANGES_REQUESTED** [review]
Changes requested.

File: components/host-dashboard/reports-section/helpers.js (hunk 1 above)
Comment (SudharakaP) [reply components/host-dashboard/reports-section/helpers.js]: Sorry about this, I thought in my mind I updated the usages; should have double checked it better. 🤯

**Comment by SudharakaP** [comment]
> Changes requested.

Sorry about the missing changes, have updated and tested; should be good now. 👍🏽

**Review by znarf — APPROVED** [review]

**Pull request merged**